includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -420,7 +418,7 @@ public function getData() {
 		$data['follow_us'] = $this->getFollowUs();
 		$data['community'] = $this->getCommunity();
 
-		if ( $this->lang === static::DEFAULT_LANG ) {
+		if ( $this->lang === self::DEFAULT_LANG ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)